packages/shared/meta/types.ts

```diff
@@ -17,7 +17,7 @@ export type MetaConfiguration = {
   pickRealmAlgorithmConfig?: AlgorithmChainConfig
   bannedUsers: BannedUsers
   synapseUrl: string
-  world: WorldConfig
+  world: any
```

Comment:
This is the only questionable line of the PR. Could you add a rationale?

Reply:
In the json, the world object can have more fields, but those fields are only going to be needed renderer side, so to avoid having to create a new PR on Kernel every time a new field is added to the json file we treat this as `any` and pass it over directly to Unity, where it will be statically typed as needed. If we don't do this, the extra fields are lost because of the explicit typing.

To avoid treating this as `any` everywhere, I added a `getWorldConfig()` selector so we can still have static typing through the `WorldConfig` type, with only for the subset fields that are used by Kernel.

Reply:
Ok, I ran this through fiddle and I think the `any` type is not needed, but I'll have to double check /shrug

Reply:
Everything works fine with the static typing, done.